addons/mail/static/src/discuss/web/avatar_card/avatar_card_popover.xml

```diff
@@ -17,13 +17,13 @@
                             <i t-elif="!user.im_status" class="fa fa-fw fa-question-circle" title="No IM status available"/>
                         </span>
                     </span>
-                    <div class="d-flex flex-column o_card_user_infos">
+                    <div class="d-flex flex-column o_card_user_infos overflow-hidden">
                         <span class="fw-bold" t-esc="user.name"/>
                         <t t-if="userInfoTemplate" t-call="{{userInfoTemplate}}"/>
-                        <a t-if="email" t-att-href="'mailto:'+email">
+                        <a t-if="email" t-att-href="'mailto:'+email" class="text-nowrap text-truncate">
```

Comment:
`text-truncate` already contains a `white-space: nowrap`. Maybe we could remove `text-nowrap` ?

![Screenshot 2024-05-14 at 09 35 54](https://github.com/odoo/odoo/assets/110090660/227b9969-7a80-443a-94fd-d1206a4ca1cb)